Content/default/.config/dotnet-tools.json

```diff
@@ -13,6 +13,12 @@
       "commands": [
         "fable"
       ]
+    },
+    "fantomas-tool": {
```

Comment:
Should there also be a corresponding FAKE target to format the sources?
What's the preferred way of referencing Fantomas from FAKE - via dotnet tool or NuGet package?

Reply:
The best approach of late is to use the .NET tool via FAKE.
For example:
https://github.com/fsprojects/fantomas/blob/ed7a94071cc7deeba849ec7806d67801094b37e6/build.fsx#L376-L409

I can add those targets if you like.

Reply:
Yes, I have given that some thought and I will pick this up someday as part of the next major version.
No real timeline for that yet though.